dirty_cat/_minhash_encoder.py

```diff
@@ -155,6 +162,41 @@ def get_fast_hash(self, string: str) -> np.array:
                 ]
             )
 
+    def compute_hash(self, x) -> np.array:
```

Comment:
Type hint the variable (should also be renamed, `x` is not descriptive), fix return type (`np.array` is a function, `np.ndarray` is the type it returns), and make the method private.
```suggestion
    def _compute_hash(self, x: str) -> np.ndarray:
```